index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>capstone project module 1</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+  <link rel="stylesheet" href="./assets/css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;200;300;400;700;900&display=swap" rel="stylesheet">
+  <script src="./scripts/index.js" type="module"></script>
+</head>
+
+<body>
+  <header class="navbar shadow-bottom">
+    <div class="dark-nav padding-x flex desktop">
+      <ul class="dark-links flex">
+        <li class="dark-link"><i class="fab fa-facebook"></i></li>
+        <li class="dark-link"><i class="fab fa-twitter"></i></li>
+        <li class="dark-link">English</li>
+        <li class="dark-link">My page</li>
+        <li class="dark-link">Logout</li>
+      </ul>
+    </div>
+    <div class="light-nav padding-x flex">
+      <button id="menu_button" class="hamburger" aria-controls="primary-navigation"><span><i
+            class="fas fa-hamburger"></i></span></button>
+      <a href="./index.html" class="home-link"><span class="logo desktop"></span></a>
+      <nav class="direction flex">
+        <ul data-visible="false" id="primary-navigation" class="links flex">
+          <li class="link"><a href="#Program">Program</a></li>
+          <li class="link"><a href="#Featured">Featured</a></li>
+          <li class="link"><a href="#partners">Sponsor</a></li>
+        </ul>
+        <a href="./about/about.html" class="btn-secondary desktop">About</a>
+      </nav>
+    </div>
+  </header>
+  <main>
+    <!-- ************************************************************************************************************************
+                                                    INTRODUCTION SECTION
+    ************************************************************************************************************************ -->
+
+    <section class="intro-section padding-x">
+      <article class="content">
+        <header class="title">
+          <h2 class="display-2 text-secondary">"Hello! Sharing world"</h2>
+          <h1 class="display-1 text-secondary">18th Latin American Congress on Free Software and Open Technologies</h1>
```

Comment:
- [ ] Please consider following the design by using the required color on the h1. This is to make sure that you follow the design guideline and replicate the design to look almost identical to the design guidelines. 

Design guideline:
<img width="406" alt="Loom_Bzg357bceb" src="https://user-images.githubusercontent.com/81468675/147156690-23809f0d-335c-470e-94d2-59b153df040d.png">

Yours:
<img width="518" alt="Loom_qR2CRu2l3C" src="https://user-images.githubusercontent.com/81468675/147156730-6e900ec7-ee01-40c7-ad8b-03831a1d6d6e.png">